runner/runners/invoke.go

```diff
@@ -200,7 +200,12 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 			}
 			if err := <-checkoutCh; err != nil {
 				log.Errorf("Checkout errored: %s", err)
+				// If there was an error there should be no lingering gitdb locks, so return
+				// In addition, co should be nil, so failing to return and calling co.Release()
+				// will result in a nil pointer dereference
+				return
```

Comment:
What does this end up returning - should we make a status struct to return here?

Reply:
still returns the AbortStatus declared below - this just breaks the goroutine early

Reply:
ah - yep, misunderstood this block, thanks!